src/httprpc.cpp

```diff
@@ -66,6 +66,8 @@ class HTTPRPCTimerInterface : public RPCTimerInterface
 static std::string strRPCUserColonPass;
 /* Stored RPC timer interface (for unregistration) */
 static std::unique_ptr<HTTPRPCTimerInterface> httpRPCTimerInterface;
+/* RPC CORS Domain, allowed Origin */
+static std::string strRPCCORSDomain;
```

Comment:
Let's call this `str_rest_cors_domain` ([new naming convention](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#variable-names)) and the parameter `-restcorsdomain` (since it's only used with `-rest=1`).

Reply:
The other options still use the old `-rpc...` naming though, so I'm not convinced this helps anyone. Even the variables declared right before `strRPCCORSDomain` use the old naming.

Reply:
I find the use of `-rpc` confusing when a setting is about the REST interface. `-rpcauth`, `-rpcuser` and `-rpcpassword` is shared between the regular RPC and REST so there it makes sense (although I'm not a fan of that sharing). The rest is just internal variables.

Reply:
REST doesn't have authentication, I thought?

Reply:
@sipa it does, see my command-line example above. It has full wallet access too.

Reply:
@Sjors I'm very confused. The REST interface only exposes public information (blocks, transactions with txindex, headers, chaininfo, utxoset, mempool), and should never require authentication. Your example seems to just use the RPC interface.